runtime/crypto_test.go

```diff
@@ -30,35 +31,6 @@ import (
 	"github.com/onflow/cadence/runtime/tests/utils"
 )
 
-func TestRuntimeCrypto_import(t *testing.T) {
```

Comment:
This is a test for old behaviour which is removed now. It also should have just been a checker test